### PR TITLE
Add chromium TypeScript definition file

### PR DESCRIPTION
**Pull request by wongpratan**

### CODE REVIEW

File: public/cli/lib/rendering/recordFrames.ts [hunk 1]
```diff
@@ -12,12 +12,10 @@ export async function record(
   logEachFrame = false,
   notifyEvent?: EventEmitter
 ) {
-  // chromium.path may or may provide a path in an asar archive.  If it does
-  // it is unusable, and we'll attempt to swap it out for the un-archived version
-  const chromiumPath = chromium.path.replace('app.asar', 'app.asar.unpacked');
-
   const browser = await puppeteer.launch({
-    executablePath: chromiumPath,
+    // chromium.path may or may provide a path in an asar archive.  If it does
```
Comment (irahopkinson) [inline public/cli/lib/rendering/recordFrames.ts:16]: nit: please fix the comment by adding 'not' to read 'chromium.path may or may not provide...'

**Review by irahopkinson — CHANGES_REQUESTED** [review]
Looks good. One small commen tot fix.

**Review by irahopkinson — APPROVED** [review]
LGTM

File: public/cli/lib/rendering/recordFrames.ts (hunk 1 above)
Comment (wongpratan) [reply public/cli/lib/rendering/recordFrames.ts]: Thank you.

**Pull request merged**